scripts/bootstrap-dev-debian.sh

```diff
@@ -1,36 +1,76 @@
 #!/bin/bash
 
-echo "Install packages needed for execution"
-
-sudo apt install -y \
-    gir1.2-keybinder-3.0 \
-    gir1.2-notify-0.7 \
-    gir1.2-vte-2.91 \
-    libkeybinder-3.0-0 \
-    python3 \
-    python3-cairo \
-    python3-dbus \
-    python3-gi \
-    python3-pbr \
-    python3-pip \
-
-echo "Install needed development packages on a Debian/Ubuntu system"
-sudo apt install -y \
-    aspell-fr \
-    colortest \
-    dconf-editor \
-    gettext \
-    glade \
-    poedit \
-    gnome-tweak-tool \
-    gsettings-desktop-schemas \
-    make \
-    pandoc \
-
-if [[ $1 == "--with-optional" ]]; then
+RUN=0
+MAKE=0
+DEV=0
+OPT=0
+ARGC="$#"
 
+while test $# -gt 0
+do
+    case "$1" in
+        run)
+          RUN=1
+          ;;
+        make)
+          MAKE=1
+          ;;
+        dev)
+          DEV=1
+          ;;
+        opt)
+          OPT=1
+          ;;
+    esac
+    shift
+done
+
+if [[ "$ARGC" == "0" ]]; then
+          RUN=1;
+          MAKE=1;
+          DEV=1;
+          OPT=0;
+fi
+
+
+if [[ $RUN == "1" ]]; then
+    echo "Install packages needed for execution"
     sudo apt install -y \
```

Comment:
@gsemet do you need the -y flag or is it just for travis? If it is just for travis could we remove the flag from this file and add `yes | sudo ./scripts/bootstrap-dev-debian.sh` to .travis.yml.

Reply:
oh and by the way same question for sudo.

Reply:
I am ok if you update the Travis script as well. I have no problem with sudo in a script.

Reply:
Ok I will push an update to this pr tomorrow.